libs/clients/middlewares/src/lib/enhanced-fetch.ts

```diff
@@ -0,0 +1,184 @@
+import CircuitBreaker from 'opossum'
```

Comment:
nit-picky, but the file name breaks the IslandIs file naming conventions:
`enhanced-fetch.ts` -> `enhancedFetch.ts`

Reply:
I love nit-picks. Guess I forgot what the naming convention was, as I've gotten so used to the nx/nest style `some-stuff.service.ts` file names. Will fix.